app/views/prisoners/show.html.erb

```diff
@@ -80,7 +80,7 @@
 
 <%= render 'prisoner_information' %>
 <%= render partial: 'shared/offence_info', locals: { editable_thd: true}  %>
-<% if @prisoner.current_parole_record.present? %>
+<% if @prisoner.tariff_date.present? || @prisoner.parole_eligibility_date.present? || @prisoner.current_parole_record.present? %>
```

Comment:
If you could shove these 3 conditions into a method on MpcOffender, you could reuse it in several places instead of 3 separate conditions. Tidier views and less error-prone future modification.